src/api/resolvers/Scalars.ts

```diff
@@ -1,6 +1,6 @@
 import { DateTime } from 'luxon';
 import { GraphQLScalarType } from 'graphql';
-import GraphQLJSON from 'graphql-type-json';
+import * as GraphQLJSON from 'graphql-type-json';
```

Comment:
I think this is functionally equivalent but worth confirming. Needed to do this to make TS happy.